src/components/CommentCard.jsx

```diff
@@ -0,0 +1,19 @@
+import React from "react";
+
+function CommentCard ({comment}){
+
+    let votes = comment.votes
+    
+    let voteType = votes < 0 ? "dislikes" : "likes";
```

Comment:
I think it's a bit confusing calling the votes "dislikes" if they're negative
Because wouldn't -1 dislikes be a good thing?
<img width="262" alt="image" src="https://github.com/Cogger101/feNcNews/assets/151947206/43ca81de-ac7f-44fc-972d-2ec4f2e7b2da">